src/lambdas/manual-change-detector/index.ts

```diff
@@ -0,0 +1,84 @@
+import SlackWebhook from 'slack-webhook'; //this is the only external dependency
+import IAM from 'aws-sdk/clients/iam';
+import zlib from 'zlib';
+import {
+    AWSEvent,
+    AWSCloudwatchEvent,
+    EventInfo,
+} from "./types";
+
+declare const Buffer;
+
+export async function handler(event: AWSEvent): Promise<void> {
+    const iam: IAM = new IAM();
```

Comment:
TypeScript can infer the types based on the assigned value.
```suggestion
    const iam = new IAM();
```